superset/dashboards/api.py

```diff
@@ -284,10 +284,9 @@ def ensure_thumbnails_enabled(self) -> Optional[Response]:
 
     def __repr__(self) -> str:
         """Deterministic string representation of the API instance for etag_cache."""
-        return "Superset.dashboards.api.DashboardRestApi@v{}{}".format(
```

Comment:
We currently use `pyupgrade` and per [here](https://github.com/asottile/pyupgrade/blob/main/README.md#f-strings) it mentions (rightfully so in my mind) that,

> pyupgrade is intentionally timid and will not create an f-string if it would make the expression longer or if the substitution parameters are sufficiently complicated (as this can decrease readability).

and thus the previous logic is likely more readable and doesn't require single use variables. I sense it's cleaner/clearer to add a `pylint: disable=consider-using-f-string`.